helpers/custom_error_page.py

```diff
@@ -0,0 +1,35 @@
+"""
+Custom error page generator.
+
+Tempesta administrator can specify files with page bodies for error responses
+generated by Tempesta FW.
+"""
+from helpers import tf_cfg
+
+__author__ = "Tempesta Technologies, Inc."
+__copyright__ = "Copyright (C) 2023 Tempesta Technologies, Inc."
+__license__ = "GPL2"
+
+
+class CustomErrorPageGenerator(object):
+    def __init__(self, data, f_path=None, default=False):
+        workdir = tf_cfg.cfg.get("General", "workdir")
+        self.f_path = f_path if f_path else workdir + "/tempesta.html"
+        self.data = data
+        if default:
+            self.generate()
+
+    @staticmethod
+    def __write(path, data):
+        with open(path, "wt") as fdesc:
```

Comment:
Why don’t we use the `tempfile` build-in library to create temporary files and directories? It looks like we are affecting the global state and thus breaking the isolation of the tests. We didn’t clean up `tempesta.html` after running the tests.

Reply:
I think we should not use `tempfile` library here because  we might need to see file after test. And also `path` is directory in `/tmp/` by default and it will be clean after reboot.